media/server/gstplayer/include/GstMimeMapping.h

```diff
@@ -81,12 +81,10 @@ convertFromCapsVectorToMimeSet(const std::vector<GstCaps *> &supportedCaps,
          {m_gstWrapper->gstCapsFromString("video/x-av1"), {"video/x-av1"}},
          {m_gstWrapper->gstCapsFromString("video/x-h264"), {"video/h264"}},
          {m_gstWrapper->gstCapsFromString("video/x-h265"), {"video/h265"}},
-         {m_gstWrapper->gstCapsFromString("video/x-vp9"), {"video/x-vp9"}},
```

Comment:
This change will disable VP9 on all platforms.
Additionally, Rialto will only return that VP9 is supported if one of the platform's GST decoders or sinks return it is supported. 
If GStreamer says VP9 is supported but it doesn't actually work, it is not Rialto's problem and workarounds for that problem shouldn't be applied in Rialto.
If there's no other way, I would suggest to use bitbake patches for the affected platform only.